uber/models.py

```diff
@@ -745,7 +745,7 @@ def assign_badges(self, group, new_badge_count, new_badge_type=c.ATTENDEE_BADGE,
 
             if int(new_badge_type) in c.PREASSIGNED_BADGE_TYPES and c.AFTER_PRINTED_BADGE_DEADLINE and diff > 0:
                 return 'Custom badges have already been ordered, so you will need to select a different badge type'
-            elif diff > 0:
+            elif diff > 0 and group.cost != 0:
```

Comment:
I think we should also add ``and not group.auto_recalc`` because otherwise we would end up not charging for additional badges to groups which SHOULD get charged for them if that group just happened to currently have a cost of 0.  (This happens for things like MIVS groups, where the initial badges are free but where we want to charge for additional badges that get added.)